modules/component-store/src/component-store.ts

```diff
@@ -51,6 +52,7 @@ export class ComponentStore<T extends object> implements OnDestroy {
 
   private readonly stateSubject$ = new ReplaySubject<T>(1);
   private isInitialized = false;
+  private notInitializedErrorMessage = `${this.constructor.name} has not been initialized`;
```

Comment:
Should we add something on how to fix this error?

Reply:
Good idea 💡
Adjusted to "ComponentStore has not been initialized yet. Please make sure it is initialized before updating/getting."